app/src/main/java/me/alexdtat/featuredetectandroid/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package me.alexdtat.featuredetectandroid
```

Comment:
I think, the name of the main package should correspond to `applicationId` in `gradle.build` and `package` in `AndroidManifest.xml`

Reply:
Will be `me.alexdtat.featuredetectandroid` ok for whole app?

Reply:
I am not very familliar with package naming traditions. Nothing offensive was meant. :)